desktop/core/src/desktop/lib/fs/proxyfs.py

```diff
@@ -79,7 +79,8 @@ def _has_access(self, fs):
       if not filebrowser_action:
         return True
       user = rewrite_user(User.objects.get(username=self.getuser()))
-      return user.is_authenticated() and user.is_active and (is_admin(user) or not filebrowser_action or user.has_hue_permission(action=filebrowser_action, app="filebrowser"))
+      user_perm = is_admin(user) or not filebrowser_action or user.has_hue_permission(action=filebrowser_action, app="filebrowser")
```

Comment:
Please keep is_authenticated in first position (position matters in if with ands.)

Reply:
e.g.

return user.is_authenticated and user.is_active and \
  (is_admin(user) or not filebrowser_action or user.has_hue_permission(action=filebrowser_action, app="filebrowser"))

Reply:
I think this is correct (add backslash after and) please confirm @romainr because the above one showing syntax error
```
return user.is_authenticated and user.is_active and \
(is_admin(user) or not filebrowser_action or user.has_hue_permission(action=filebrowser_action, app="filebrowser"))
```

Reply:
Yes, somehow my \ disappeared in my comment, also add +2 indent on second line